docs/testing.md

```diff
@@ -240,7 +240,7 @@ Both widgets now detect the Rails application environment automatically -- simpl
 
 ### Testing locally
 
-There is a simple web app in our repository called [web-overlay-test](https://github.com/gumroad/web-overlay-test). Once you check it out, follow these instructions to test the overlay and/or embed.
+There is a simple web app in our repository called [web-overlay-test](https://github.com/antiwork/web-overlay-test). Once you check it out, follow these instructions to test the overlay and/or embed.
```

Comment:
Likely a private repo, might need to make it public too

Reply:
Probably good to open source and remain gumroad-overlay-test